app/src/main/res/values/strings.xml

```diff
@@ -249,8 +249,6 @@
 
     <!-- This is the prompt when autofill is triggered and the application is locked. %1$s will be replaced with application name -->
     <string name="autofill_authenticate_cta">Unlock %1$s</string>
-    <!-- This is the error message when autofill is triggered but no domain is detected. This is unlikely. -->
-    <string name="autofill_error_no_hostname">Unexpected hostname format</string>
     <!-- This is the prompt when autofill is triggered, application is unlocked, and no passwords are detected. %1$s will be replaced with application name. -->
     <string name="autofill_search_cta">Search %1$s</string>
```

Comment:
Is there a better way to describe what "Search Lockwise" actually does? This sounds like something that could mean "search for" or "search in" or "search with".